devicetypes/smartthings/zwave-lock-without-codes.src/zwave-lock-without-codes.groovy

```diff
@@ -405,6 +405,10 @@ private def handleBatteryAlarmReport(cmd) {
 	def deviceName = device.displayName
 	def map = null
 	switch (cmd.zwaveAlarmEvent) {
+		case 0x01: //power has been applied, check if the battery level updated
+			runIn(10, "queryBattery", [overwrite: true, forceForLocallyExecuting: true])
+			result << response(secure(zwave.batteryV1.batteryGet()))
+			break;
 		case 0x0A:
```

Comment:
Just curious, does case 130 not apply for this DTH as it does for `zwave-lock.groovy`?

Reply:
I don't believe any of the locks on this DTH use the OEM/alarmv1 codes.